src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputEmptyDefault.java

```diff
@@ -1,89 +0,0 @@
-public class InputEmptyDefault {
```

Comment:
Why is this file deleted?
We expect file to be moved, with just changing package and changing class name.